apps/site/lib/site_web/templates/partial/_hidden_icons.html.eex

```diff
@@ -57,4 +57,7 @@
   <div id="icon-feature-map-station-marker">
     <%= svg("icon-map-station-marker.svg") %>
   </div>
+  <div id="icon-feature-silver_line">
+    <%= svg("icon-silver-line-default.svg") %>
+  </div>
```

Comment:
Right now the `for mode <- [:subway, :commuter_rail, :bus, :ferry, :silver_line] do` loop is outputting another `<div id="icon-feature-silver_line">` with a bus icon inside, and since that's earlier up the template that one's getting picked up first. So you've got two options here -- you can leave this here and remove `:silver_line` from the loop, or remove this HTML and fix the `mode_icon` function above.